docs/user_guide/workloads/run_mpi.md

```diff
@@ -0,0 +1,106 @@
+---
+id: run_mpi
+title: Run MPI Jobs
+description: How to run MPI jobs with YuniKorn
+keywords:
+ - mpi
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+
+# MPI Operator
+
+For more details, please read https://github.com/kubeflow/mpi-operator#installation for deploying this operator.
+
+# Pure MPI example
+
+This example shows to run a pure MPI application.
+
+The program prints some basic information about the workers.
+Then, it calculates an approximate value for pi.
+
+Here is a Pi yaml example [example](https://github.com/apache/yunikorn-k8shim/blob/master/deployments/examples/mpioperator/Pi/pi.yaml).
+```yaml
+apiVersion: kubeflow.org/v2beta1
+kind: MPIJob
+metadata:
+  name: pi
+spec:
+  slotsPerWorker: 1
+  runPolicy:
+    cleanPodPolicy: Running
+    ttlSecondsAfterFinished: 60
+  sshAuthMountPath: /home/mpiuser/.ssh
+  mpiReplicaSpecs:
+    Launcher:
+      replicas: 1
+      template:
+        labels:
+          applicationId: "mpi_job_pi"
+          queue: root.mpi
+        spec:
+          schedulerName: yunikorn
+          containers:
+          - image: mpioperator/mpi-pi
+            name: mpi-launcher
+            securityContext:
+              runAsUser: 1000
+            command:
+            - mpirun
+            args:
+            - -n
+            - "2"
+            - /home/mpiuser/pi
+            resources:
+              limits:
+                cpu: 1
+                memory: 1Gi
+    Worker:
+      replicas: 2
+      template:
+        labels:
+          applicationId: "mpi_job_pi"
+          queue: root.mpi
+        spec:
+          schedulerName: yunikorn
+          containers:
+          - image: mpioperator/mpi-pi
+            name: mpi-worker
+            securityContext:
+              runAsUser: 1000
+            command:
+            - /usr/sbin/sshd
+            args:
+            - -De
+            - -f
+            - /home/mpiuser/.sshd_config
+            resources:
+              limits:
+                cpu: 1
+                memory: 1Gi
+```
+```
```

Comment:
Add '## Run MPI Jobs' to separate the pi.yaml and the kubectl command.

Reply:
As @0yukali0 mentioned: yaml and kubectl separation is needed to clarify what is documented. You can use '## Example MPI job yaml` and '## Run MPI Jobs'

Reply:
Following the tensor flow docs, I notice you just added a simple text to separate the yaml from the kubectl code.  I figured that would be fine here.  

If you want a section, I can add that also.